grails-app/services/au/org/ala/profile/hub/ExportService.groovy

```diff
@@ -126,25 +123,140 @@ class ExportService {
             withPool(THREAD_COUNT) {
                 children.eachParallel {
                     if (it.profileId != params.profileId && it.scientificName != params.profileId) {
-                        model.profiles << loadProfileData(it.profileId, model.opus, params)
+                        curatedModel.profiles << loadProfileData(it.profileId, opus, params, latest)
                     }
                 }
             }
 
-            model.profiles = model.profiles.sort { it.profile.scientificName }
+            curatedModel.profiles = curatedModel.profiles.sort { it.profile.scientificName }
         }
 
-        wkhtmltoxService.makePdf(
-                view: "/pdf/_profile",
-//                footer: "/pdf/_pdfFooter",
-//                header: "/pdf/_pdfHeader",
-                model: model,
-                marginLeft: 15,
-                marginTop: 20,
-                marginBottom: 20,
-                marginRight: 15,
-                headerSpacing: 10,
-        )
+        curatedModel.options << [
+                allowFineGrainedAttribution: opus.allowFineGrainedAttribution,
+                displayToc: curatedModel.profiles?.size() > 1
+        ]
+
+        curatedModel << [
+                cover: [
+                        title       : opus.title,
+                        subtitle    : curatedModel.profiles[0]?.profile?.fullName,
+                        logo        : opus.logoUrl,
+                        banner      : opus.bannerUrl,
+                        primaryImage: curatedModel.profiles[0]?.profile?.primaryImage?: (curatedModel.profiles[0]?.profile?.images?.size() > 0 ? curatedModel.profiles[0]?.profile?.images[0].leftImage.largeImageUrl : '')
+                ],
+                colophon: [
+                        copyright: opus.copyrightText,
+                        logo        : opus.logoUrl,
+                        profileLink: "${grailsApplication.config.grails.serverURL}/opus/${params.opusId}/profile/${curatedModel.profiles[0]?.profile?.uuid}"
```

Comment:
params.opusId may be the opus shortname, which can be edited. For the PDF, it's probably safer to always use the uuid.